test/stats_test.go

```diff
@@ -32,15 +33,15 @@ import (
 )
 
 // TestPeerForClientStatsHandler configures a stats handler that
-// verifies that peer is sent for OutPayload, InPayload, End
-// stats handlers.
+// verifies that peer is sent all stats handler callouts instead
+// of begin and pickerUpdated
 func (s) TestPeerForClientStatsHandler(t *testing.T) {
-	statsHandler := &peerStatsHandler{}
+	psh := &peerStatsHandler{}
 
-	// Define expected stats callouts and whether a peer object should be populated.
+	// Stats callouts & peer object population.
 	// Note:
-	//   * Begin stats don't have peer information as the RPC begins before peer resolution.
-	//   * PickerUpdated stats don't have peer information as the picker operates without transport-level knowledge.
+	// * Begin stats lack peer info (RPC starts pre-resolution).
+	// * PickerUpdated: no peer info (picker lacks transport details).
```

Comment:
No you can keep the full comment, just wrap it to 80 characters. Just make sure it wraps 80 by 80.

Reply:
done